src/components/bs5/footer/footer.data.json

```diff
@@ -32,15 +33,14 @@
     },
     "acknowledgements": [
         {
-            "title": "",
-            "content": "<p>We pay our respects to the Aboriginal and Torres Strait Islander ancestors of this land, their spirits and their legacy. The foundations laid by these ancestors—our First Nations peoples—give strength, inspiration and courage to current and future generations towards creating a better Queensland.</p>"
```

Comment:
Content to return to previous as per this figma: https://www.figma.com/design/nMmoukrjPJ8qvUiwmuj1bs/forgov.qld.gov.au?node-id=2013-93774&t=sEtDEr85FExHefxA-1

Heading: Cultural acknowledgement
Content: We pay our respects to the Aboriginal and Torres Strait Islander ancestors of this land, their spirits and their legacy. The foundations laid by these ancestors—our First Nations peoples—give strength, inspiration and courage to current and future generations towards creating a better Queensland.

Reply:
Hi Katja, I have updated the default AoC statement on json data.

Updated Storybook link is:
https://660e3e06af8da379e06be302-zqkmtfxvkk.chromatic.com/?path=/story/core-footer--dark